lib/actions/tunnel.ts

```diff
@@ -0,0 +1,152 @@
+/*
+Copyright 2016-2017 Balena
```

Comment:
Presumably this should be 2019? Either way, you don’t need to specify a range, just the first year the file was released under this license.

Reply:
copy-paste fail